src/ert/scheduler/scheduler.py

```diff
@@ -142,12 +148,17 @@ async def _stop_long_running_jobs(
                         > long_running_factor * self._average_job_runtime
                         and not task.done()
                     ):
-                        logger.info(
+                        logger.error(
```

Comment:
Is it considered an error? I think that this was "asked" by the user when specifying `STOP_LONG_RUNNING`

Reply:
Good question. From the perspective of the scheduler, it is not an error, I agree. From the perspective of the realization (or  job.py), it is an error. For, reasons, the corresponding logging for max_runtime is logged as an error, but this happens inside job.py which you could argue has a different perspective (the realization). 

It makes sense to have the same logleven for stop_long_running and max_runtime, and whether it is logged from scheduler.py or job.py does not matter for the user. 

Maybe warning is a fair compromise?

Reply:
I have changed both to warning.

Reply:
warning sounds good!